### PR TITLE
Update to support .NET Core 2.2 & .NET Framework 4.6.1

### PR DESCRIPTION
**Pull request by hajuuk**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +0,0 @@
-language: csharp
```
Comment (maximn) [inline .travis.yml]: Why did you delete this file? The CI can't run without it.

File: .nuspec [hunk 2]
```diff
@@ -1,29 +0,0 @@
-<?xml version="1.0"?>
```
Comment (maximn) [inline .nuspec]: Why did you delete this file? Can't publish new nuget package without it.

File: GoogleMapsApi.Test/IntegrationTests/BaseTestIntegration.cs [hunk 3]
```diff
@@ -12,39 +14,41 @@ namespace GoogleMapsApi.Test.IntegrationTests
 
     public class BaseTestIntegration
     {
-        private string apiKey;
+        private string _apiKey;
```
Comment (maximn) [inline GoogleMapsApi.Test/IntegrationTests/BaseTestIntegration.cs]: Why did you change the naming convention? In this project we use C# naming convention means private variables are camleCase. https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions

File: GoogleMapsApi.Test/packages.config [hunk 4]
```diff
@@ -1,5 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (maximn) [inline GoogleMapsApi.Test/packages.config]: Why did you delete it?

File: GoogleMapsApi/Engine/MapsAPIGenericEngine.cs [hunk 5]
```diff
@@ -51,7 +50,7 @@ public static int HttpsConnectionLimit
 		private const string AuthenticationFailedMessage =
 			"The request to Google API failed with HTTP error '(403) Forbidden', which usually indicates that the provided client ID or signing key is invalid or expired.";
 
-		static MapsAPIGenericEngine()
+		static MapsApiGenericEngine()
```
Comment (maximn) [inline GoogleMapsApi/Engine/MapsAPIGenericEngine.cs]: This is a breaking change! Anyway, it doesn't relate to the .NET Core upgrade.

File: MapsApiTest/Program.cs [hunk 6]
```diff
@@ -15,31 +15,31 @@
...
-			{
-				Origin = "New York",
-				Destination = "Queens",
-				TravelMode = TravelMode.Transit,
-				DepartureTime = DateTime.Now
-			};
-
-			DirectionsResponse transitDirections = GoogleMaps.Directions.Query(transitDirectionRequest);
-			PrintDirections(transitDirections);
+    class Program
```
Comment (maximn) [inline MapsApiTest/Program.cs]: Please don't change the spacing.

**Comment by hajuuk** [comment]
I do apologise for the PR, I will submit a new one later tonight.

**Comment by hajuuk** [comment]
I'll reply some comment that is required by the update. The rest I will not submit in new PR.

File: GoogleMapsApi.Test/IntegrationTests/BaseTestIntegration.cs (hunk 3 above)
Comment (hajuuk) [reply GoogleMapsApi.Test/IntegrationTests/BaseTestIntegration.cs]: The Visual Studio 2019 don't really need the file for package to nuget format. It is included in the csproj file.

File: MapsApiTest/Program.cs (hunk 6 above)
Comment (hajuuk) [reply MapsApiTest/Program.cs]: Would not do that in new PR.

File: GoogleMapsApi.Test/packages.config (hunk 4 above)
Comment (hajuuk) [reply GoogleMapsApi.Test/packages.config]: The Visual Studio 2019 just included thus file into csproj file.

File: .travis.yml (hunk 1 above)
Comment (hajuuk) [reply .travis.yml]: Will not touch this file in new PR

File: GoogleMapsApi.Test/IntegrationTests/BaseTestIntegration.cs (hunk 3 above)
Comment (hajuuk) [reply GoogleMapsApi.Test/IntegrationTests/BaseTestIntegration.cs]: Would not touch naming as well.

File: .nuspec (hunk 2 above)
Comment (hajuuk) [reply .nuspec]: The Visual Studio 2019 don't really need the file for package to nuget format. It is included in the csproj file.

File: GoogleMapsApi/Engine/MapsAPIGenericEngine.cs (hunk 5 above)
Comment (hajuuk) [reply GoogleMapsApi/Engine/MapsAPIGenericEngine.cs]: Would not touch it as well.

File: .nuspec (hunk 2 above)
Comment (maximn) [reply .nuspec]: Since I wrote this lib I moved to mac and don't have visual studio installed so please keep this file because I'm using myget service to publish new versions.
